bootstrap/kubeadm/internal/cloudinit/boot_commands.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2025 The Kubernetes Authors.
```
Probably a nitpick, but it looks a bit outdated.